src/main/java/seedu/address/logic/parser/AddPersonCommandParser.java

```diff
@@ -17,20 +17,20 @@
 /**
  * Parses input arguments and creates a new AddCommand object
```

Comment:
Perhaps the documentation should be changed to reflect the change of class name as well. In this case, `AddCommand` should be `AddPersonCommand`.

Reply:
Good point! Will change!